ASPNetCoreMastersTodoList.Api/ApiModels/ItemUpdateBindingModel.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ASPNetCoreMastersTodoList.Api.ApiModels
+{
+    public class ItemUpdateBindingModel
+    {
+        [Required]
```

Comment:
sir I think we don't need the required validation attribute here since we have it on the route template for the HttpPut in the ItemsController. Can test and verify?